debian/control

```diff
@@ -12,7 +12,8 @@ Vcs-Browser: https://github.com/dlech/keepasshttp
 
 Package: keepass2-plugin-keepasshttp
 Architecture: all
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, libmono-system-xml-linq4.0-cil, libmono-system-data-datasetextensions4.0-cil, libmono-system-runtime-serialization4.0-cil, libnewtonsoft-json5.0-cil
+Recommends: keepass2
```

Comment:
Actually, `keepass2` should probably be under the Depends: section since this is useless without it.